internal/pkg/dl/agent.go

```diff
@@ -80,7 +80,7 @@ func FindOfflineAgents(ctx context.Context, bulker bulk.Bulk, policyID string, u
 	}
 
 	if len(res.Hits) == 0 {
-		return nil, ErrNotFound
+		return nil, nil
```

Comment:
[Blocker]
I believe it's better do not change the logic here. Also because this pattern is used across the package, thus changing it here would make this method not consistent with the rest of the package.

It's better the caller to test for `ErrNotFound` and handle it properly

Reply:
fixed